package.json

```diff
@@ -23,10 +23,10 @@
   ],
   "scripts": {
     "size": "size-limit",
-    "test": "standard && ava",
```

Comment:
Don't remove the `test` command (this is still used locally).

Reply:
Rectified.